server/telemetry.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+const (
+	telemetryStartSourceWebapp  = "webapp"
+	telemetryStartSourceCommand = "command"
+)
+
+func (p *Plugin) trackConnect(userID string) {
+	p.tracker.TrackUserEvent("connect", userID, map[string]interface{}{})
+}
+
+func (p *Plugin) trackDisconnect(userID string) {
+	p.tracker.TrackUserEvent("disconnect", userID, map[string]interface{}{})
+}
+
+func (p *Plugin) trackMeetingStart(userID, source string) {
+	p.tracker.TrackUserEvent("start_meeting", userID, map[string]interface{}{
```

Comment:
Would `meeting_started` work better to match the other keys?  (ex. meeting_duplicated, meetins_forced)